src/discord/commands/moderation/nickname.ts

```diff
@@ -0,0 +1,40 @@
+import { Command, AccessLevel, CommandError, Logger, Guards } from "dd-botkit";
+
+export const NicknameCommand: Command = {
+    opts: {
+        name: "nickname",
+        access: AccessLevel.MODERATOR,
+        category: "Moderation",
+        guards: [
+            Guards.Argumented(
+                "nickname",
+                "Force changes the nickname of a user",
+                [
+                    {
+                        name: "user",
+                        type: "user",
+                        required: true
+                    },
+                    {
+                        name: "nickname",
+                        type: "string",
+                        required: true
+                    }
+                ]
+            )
+        ]
+    },
+    handler: async (msg, next) => {
+        const [user, nickname] = msg.args;
+        const member = msg.guild.members.get((args[0] as User).id);
+
+
+        if (!member) {
+            throw new CommandError({
+                message: "Please specify a member of this guild."
+            });
+        }
+
+        await member.setNickname(nickname);
```

Comment:
It is best practice for BotKit commands to give some indication of the status of the command. For example, you could do `await msg.success()` which will react to the original command with a 👌.

Reply:
Next thing I was going to implement was a msg.react. Thanks for letting me know, I’ve changed the code and put that in there.